circle.sh

```diff
@@ -41,6 +41,16 @@ case "$1" in
 
     case $CIRCLE_NODE_INDEX in
       0)
+        # We need to build the helios-solo image because we use TemporaryJobs in helios-testing
+        # tests, and TemporaryJobs uses the helios-solo container.
+        sudo apt-get install -y jq
+        mvn -P build-images -P build-solo package -DskipTests=true -Dmaven.javadoc.skip=true \
+          -B -V -pl helios-services
+
+        # tag the helios-solo image we just built
+        solo_image=$(cat helios-services/target/test-classes/solo-image.json | jq -r '.image')
+        docker tag -f $solo_image spotify/helios-solo:latest
```

Comment:
this looks like duplication from what is under the `5)` branch

1. if it is really necessary to duplicate, can these commands be moved to a function?
2. won't there be a race between building and tagging the image as `spotify/helios-solo:latest`
3. would it be simpler to move the tests that need this image to be executed by the `5)` node?